scripts/configure-vscode.sh

```diff
@@ -23,6 +23,7 @@ cat >.vscode/settings.json.new <<\EOF ||
   "editor.defaultFormatter": "esbenp.prettier-vscode",
 
   "typescript.preferences.importModuleSpecifierEnding": "js",
+  "typescript.tsdk": "node_modules/typescript/lib",
```

Comment:
What difference does this make? Was your VSC version too far ahead?

Reply:
I just remember needing to advise some folks to "Select Typescript Version" to point to the SDK repo in the past in order to fix discrepancies between VSCode and `tsc`.